### PR TITLE
[W16-4] CLInic

### PR DESCRIPTION
**Pull request by ChesterSng**
@blewjy 
@clawyq 
@syahiranrafi
@99percentile

### CODE REVIEW

**Comment by bqnguyen94** [comment]
@ChesterSng was this PR created by mistake?

**Comment by ChesterSng** [comment]
@bqnguyen94 I think it is one of the procedures for documentation

> One member: create a PR from your team repo master branch to [nus-cs2103-AY1819S1/addressbook-level4] master branch. PR name: [Team ID] Product Name e.g., [T09-2] Contact List Pro.

**Comment by bqnguyen94** [comment]
@ChesterSng Uhm isn't the PR supposed to update your team's documentation?

**Comment by blewjy** [comment]
@bqnguyen94 Yes we thought we'd just PR first then update the documentation along the way. We will update the documentations before the this week's tutorial.

**Comment by bqnguyen94** [comment]
@ChesterSng Let me know when the PR is ready for review :)

File: README.adoc [hunk 1]
```diff
@@ -15,19 +15,13 @@ ifndef::env-github[]
...
+* This is a desktop Clinic Address Book application. It has a GUI but most of the user interactions happen using a CLI (Command Line Interface).
+* It is a Java application intended for use of clinics to store patient information, medicine stock levels and to manage patient queues.
 
 == Site Map
 
 * <<UserGuide#, User Guide>>
 * <<DeveloperGuide#, Developer Guide>>
-* <<LearningOutcomes#, Learning Outcomes>>
 * <<AboutUs#, About Us>>
 * <<ContactUs#, Contact Us>>
```
Comment (bqnguyen94) [inline README.adoc:27]: Change this page's title, team name, and add `addressbook-level4` to acknowledgement as well :)

File: docs/AboutUs.adoc [hunk 2]
```diff
@@ -4,53 +4,54 @@
...
+=== Syahiran Rafi
+image::syahiranrafi.jpg[width="150", align="left"]
+{empty}[http://github.com/syahiranrafi[github]] [<<syahiranrafi#, portfolio>>]
 
 Role: Developer +
 Responsibilities: UI
 
 '''
+
+==Travis
```
Comment (bqnguyen94) [inline docs/AboutUs.adoc:57]: Unintended?

**Review by bqnguyen94 — COMMENTED** [review]
Nice mockup 👍

Note: don't close this PR. I will use it to check your progress in subsequent weeks :)

**Comment by ChesterSng** [comment]
@bqnguyen94 Noted on the comments!

File: README.adoc [hunk 3]
```diff
@@ -1,4 +1,4 @@
-= Address Book (Level 4)
+= CLInic
 ifdef::env-github,env-browser[:relfileprefix: docs/]
 
 https://travis-ci.org/se-edu/addressbook-level4[image:https://travis-ci.org/se-edu/addressbook-level4.svg?branch=master[Build Status]]
```
Comment (bqnguyen94) [inline README.adoc]: Please update these badges to point to your repo's status

File: docs/AboutUs.adoc [hunk 4]
```diff
@@ -4,53 +4,54 @@
...
 {empty} +
 We are a team based in the http://www.comp.nus.edu.sg[School of Computing, National University of Singapore].
 
 == Project Team
 
-=== John Doe
-image::damithc.jpg[width="150", align="left"]
-{empty}[http://www.comp.nus.edu.sg/~damithch[homepage]] [https://github.com/damithc[github]] [<<johndoe#, portfolio>>]
+=== Chester Sng
+image::chestersng.jpg[width="150", align="left"]
```
Comment (bqnguyen94) [inline docs/AboutUs.adoc]: All images are not displaying on netlify?

**Review by bqnguyen94 — COMMENTED** [review]
Good progress 👍 a few more things to note:

* Update the repo's description on Github (right below `Code` `Issues` `Pull Requests` etc.). Right now it is saying `Address Book sample application (Level 4) https://se-edu.github.io/addressbook-…`
* Update netlify-generated page's nav bar according to your application
* `UserGuide`: update author and introduction sections, and add ‘coming in v2.0’ for features not yet implemented

**Review by bqnguyen94 — COMMENTED** [review]
V1.2 comment:
* I see that you have updated your Developer Guide quite frequently as you progressed, good job and keep it up 👍 though, while you update the DG do remember to change/delete details specific to `AddressBook-Level4` as well, for example most urls in Section 3 still link to AB4's files, or Section 4.1 the app name is `AddressBook`
* Also, you need to update the title and badges in your README file asap. I have mentioned this and a few other things in the previous review

**Comment by blewjy** [comment]
> IntelliJ threw a warning: Found duplicated code in BrowserPanel.java.
> Anything that can be done about it?

Will be addressing this. Thanks for the heads up.

**Review by bqnguyen94 — COMMENTED** [review]
Some comments on your DG:
* I see you have updated activity diagram and sequence diagram according to your app, well done! Though figures 18, 24, 25, 26 look quite small and therefore are inconsistent with other figures and hard to read the words on them.
* A quick search pointed out that there are still lots of references to `AddressBook` that need to be changed (I assume you have renamed all components in the codebase?)
* Some diagrams still have `AddressBook` inside. Maybe omit them or do a quick fix if you have time.

**Comment by ChesterSng** [comment]
Noted Quoc Bao, thanks!

**Comment by ChesterSng** [comment]
@bqnguyen94 Hi Quoc Bao, we are not sure the travis build is failing here, our own repo is passing no problem.

**Comment by bqnguyen94** [comment]
Prof @damithc Travis is detecting trailing whitespace errors in their docs, but their latest raw version doesn't contain trailing whitespaces at the specified locations, and their repo's Travis is green. Any ideas what could be going on here?

**Comment by damithc** [comment]
It's fine for this PR to fail CI as long as the main repo is passing CI. i'll close this PR so that they can open a new PR to see if that works.

**Pull request closed without merging**